documentation/local-dev/roslyn/LocalDev.Packages.props

```diff
@@ -0,0 +1,24 @@
+<Project>
+  <!-- If \LocalDev.Packages.props exists, it will be loaded and merged with Directory.Packages.props, allowing for local adhoc build customizations. -->
+  <PropertyGroup>
+    <!-- Use the local version of Roslyn -->
+    <RoslynVersion>4.14.0-dev</RoslynVersion>
```

Comment:
Is there any way to update these versions?

Reply:
How?

Reply:
I'm not sure. I was thinking that if it's built locally, we could perhaps overwrite something in this file as part of the build process? That only really works if it's in a fixed location and we just conditionally import it rather than having it normally live in documentation as here.

Reply:
We could potentially fish out the version from the other repo, using `RoslynRoot` set in [LocalDev.Build.props](https://github.com/dotnet/sdk/pull/48739/files#diff-042c0b9453032fc0c6eb6ef7fa8cbbeb243507fed0c5ee640e388d72b053e6d3). Setting RoslynVersion in LocalDev.Packages.props is easy enough, one time step though.